test/factories/accounts/user_factory.ex

```diff
@@ -0,0 +1,15 @@
+defmodule GoogleCrawler.Accounts.UserFactory do
+  defmacro __using__(_opts) do
+    quote do
+      def user_email, do: "user#{System.unique_integer()}@example.com"
+      def user_password, do: "hello world!"
```

Comment:
Should we make these functions private?

Reply:
Fixed in https://github.com/ankitkalia1195/elixir-google-crawler/pull/34/commits/f300517565b4673169c7ca8270285d66e2c1cbfe